packages/atomic/src/locales.json

```diff
@@ -272,6 +272,30 @@
     "en": "Most recent",
     "fr": "Plus récents"
   },
+  "clearFilters": {
+    "en": "Clear {{count}} filter",
+    "fr": "Enlever {{count}} filtre"
```

Comment:
How does the plural versus count aspect work with i18n ?

Because we'd need to have `filtres` when count > 1

Same for all other strings here.

Reply:
The line underneath the one you commented `clearFilters_plural`